Shared/Tests.Shared/RelationshipTests.cs

```diff
@@ -437,6 +437,56 @@ public void Backlinks()
             Assert.That(maggie.Owners, Is.EquivalentTo(new[] { dani }));
         }
 
+        [Test]
+        public void RecursiveBacklinksBossTestOnly()
```

Comment:
I'd prefer if we could try and stick with a more spec-style naming of our tests. The fact that this involves a boss is not relevant. This test could be named `BacklinksShouldWorkRecursively`